install.sh

```diff
@@ -14,106 +34,173 @@ echo " IP-Address: $IP_RANGE.1"
 echo " IP-Range: $IP_RANGE.0"
 echo " Incomming device: $INC"
 echo " WiFi device: $WIFI"
-
-
-# update os
-apt-get update
-#apt-get -y upgrade
-apt-get -y install hostapd isc-dhcp-server iptables
-
-# check wlan0 available
-
-if ! ifconfig -a | grep "$WIFI"; then
+echo
```

Comment:
Why `echo`?

Reply:
During the begin function, the display is without new line (to permit the done to be displayed on the same line). In cas you want to display this error, I choose to display it on a single line. So I added an echo for adding newline. In case you modify the begin function and don't do an echo -n, the error will be more easy to find because, you will have a blank line to separe it from the normal flow (it could or should also be send to error flow, I should correct it).